plugins/inputs/prometheus/prometheus.go

```diff
@@ -76,16 +79,12 @@ var sampleConfig = `
   ##   example: metric_version = 1; deprecated in 1.13
   ##            metric_version = 2; recommended version
   # metric_version = 1
-
```

Comment:
Dito as well as for all those instances below.

Reply:
Fixed!